modules/unsupported/ysld/src/main/java/org/geotools/ysld/YamlObject.java

```diff
@@ -30,19 +30,18 @@
  * Base class for Yaml object wrappers.
  */
 public class YamlObject<T> {
```

Comment:
Could we get a small discussion of what YamlObject is for, it would make understanding YamlMap and friends easier:

Yaml is represented as atomic types (literals, expressions) and YamlObjects (for wrapping lists and maps). The factory method {@link #create( obj ) } will sort out the details. These YamlObjects can be used to stage data for output as Yaml.